### PR TITLE
Recreated Haxfred's personality on new system

### PR DESCRIPTION
**Pull request by crookedneighbor**

### CODE REVIEW

**Comment by crookedneighbor** [comment]
Realized that I never opened a PR for this...

File: lib/haxfred-irc-legacy-personality.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    if (haxfred.irc.client.nick !== from) {
+        haxfred.irc.say("Hey there, " + from + " welcome to " + Haxfred.config.channels[0] + "!");
+    }
+
+    deferred.resolve();
+  });
+
+  haxfred.on('irc.directMsg', '', function(data, deferred) {
+
+    fs.readFile(randomPhrases, 'utf8', function(err, data) {
```
Comment (NickTomlin) [inline lib/haxfred-irc-legacy-personality.js:66]: This is said knowing this is legacy code, but I think we should avoid reading the file on every direct message here. Perhaps just storing a reference to it on start up (when we check for it or create it)?

File: lib/haxfred-irc-legacy-personality.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    "Totally",
+                    "I see nothing unethical in the job it does. Why shouldn't you send a copy of some music to a friend?",
+                    "I'm always happy when I'm protesting.",
+                    "'Free software' is a matter of liberty, not price. To understand the concept, you should think of 'free' as in 'free speech,' not as in 'free beer'.",
+                    "For personal reasons, I do not browse the web from my computer. (I also have not net connection much of the time.) To look at page I send mail to a demon which runs wget and mails the page back to me. It is very efficient use of my time, but it is slow in real time.",
+                    "Playfully doing something difficult, whether useful or not, that is hacking.",
+                    "Copying all or parts of a program is as natural to a programmer as breathing, and as productive. It ought to be as free.",
+                    "All you need is love."];
+    file.on('error', function(err) { console.log(err);});
+    phrases.forEach(function(v) { file.write(v + '\n'); });
```
Comment (NickTomlin) [inline lib/haxfred-irc-legacy-personality.js:33]: I think we could get away with `file.write(phrases.join('\n'));` here?

File: lib/haxfred-irc-legacy-personality.js (hunk 1 above)
Comment (crookedneighbor) [inline lib/haxfred-irc-legacy-personality.js:66]: I'm :+1: on this

Comment (lejeunerenard) [inline lib/haxfred-irc-legacy-personality.js:66]: Also :+1:

File: lib/haxfred-irc-legacy-personality.js (hunk 2 above)
Comment (lejeunerenard) [inline lib/haxfred-irc-legacy-personality.js:33]: :+1:

File: lib/haxfred-irc-legacy-personality.js (hunk 1 above)
Comment (crookedneighbor) [inline lib/haxfred-irc-legacy-personality.js:66]: I remember what I was thinking here. Having it read in the file each time lets you edit the randomPhrases file without having to restart Haxfred. Not a good enough reason to keep this though, it's relatively easy to restart haxfred.

**Pull request closed without merging**